docs/loader.md

```diff
@@ -0,0 +1,11 @@
+---
+id: loader
+title: loader
+sidebar_label: Loader
+---
+
+## Loader
```

Comment:
MicroApp Loader, Web Component Loader

Reply:
Added micro app loader and web component loader